tests/test_integrations.py

```
@@ -274,3 +274,84 @@ def test_get_provider_module_invalid_raises_error(self):
+                mock_response = MagicMock()
+                mock_response.status_code = 200
+                mock_response.headers = {"content-type": "image/jpeg"}
+                mock_response.aiter_bytes.return_value = [b"data"]
+                mock_client.build_request.return_value = "request"
+                mock_client.send.return_value = mock_response
+
+                with patch("app.integrations.router._get_proxy_client", return_value=mock_client):
+                    with patch("app.core.encryption.decrypt_token", return_value="decrypted-key"):
+
+                        # Call the endpoint function directly
+                        response = await proxy_thumbnail(
```

Ruff (0.14.13)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)